### PR TITLE
Fix: Deezer albums

### PR DESCRIPTION
**Pull request by arctixdev**
Fixes Deezer album track parsing by adding required disc number

### CODE REVIEW

File: music_assistant/server/providers/deezer/__init__.py [hunk 1]
```diff
@@ -664,6 +664,7 @@ def parse_track(self, track: deezer.Track, user_country: str, position: int = 0)
             metadata=self.parse_metadata_track(track=track),
             track_number=position,
             position=position,
+            disc_number=getattr(track, "disk_number", 1),
```
Comment (marcelveldt) [inline music_assistant/server/providers/deezer/__init__.py:667]: is it really disk number (with a k) and not disc number ?

Comment (arctixdev) [reply music_assistant/server/providers/deezer/__init__.py:667]: Yup, i wondered as well :thinking: https://deezer-python.readthedocs.io/en/stable/api_reference/resources/track.html#deezer.Track.disk_number at least according to the code and the docs. But could be a typo in the original library. Let me double check

Comment (arctixdev) [reply music_assistant/server/providers/deezer/__init__.py:667]: Nope, thats also how it is written in the official deezer api documentation: https://developers.deezer.com/api/track

Comment (marcelveldt) [reply music_assistant/server/providers/deezer/__init__.py:667]: ok, good that you double checked, all good then!

**Review by marcelveldt — APPROVED** [review]

**Pull request merged**